src/components/Navbar/Navbar.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import './Navbar.css';
```

Comment:
You are still using a global css file here instead of CSS modules.
Please use `Nav.module.css` and import your styles similar to what was done in `LoginPage.tsx`

As with the CSS class names you have selected, its highly likely for there to be conflicts later on down the line